src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class 'RightTriangle' should not begin with an empty line. Please remove the empty line at the start of the class definition.